src/qibolab/instruments/rfsoc/driver.py

```diff
@@ -69,6 +71,23 @@ def connect(self):
     def disconnect(self):
         """Empty method to comply with Instrument interface."""
 
+    @staticmethod
+    def _try_to_execute(server_commands, host, port):
+        try:
+            return client.connect(server_commands, host, port)
+        except RuntimeError as e:
+            if "exception in readout loop" in str(e):
+                log.warning(
+                    "%s %s",
+                    "Exception in readout loop. Attempting again",
+                    "You may want to increase the relaxation time.",
+                )
```

Comment:
Is it possible to have a custom exception, something like `ReadoutLoopException` and catch it directly instead of using the if statement?

Reply:
Thanks for the review! Currently, exceptions are generated by qibosoq so I will have to update it an do a release to implement this. I'm going to do along with the interpolated-channel one. 
I still like the idea of the specific exception, I have opened a related PR in qibosoq: https://github.com/qiboteam/qibosoq/pull/115